src/nxdoc/nuxeo-server/administration/search-setup.md

```diff
@@ -837,14 +762,13 @@ Instead of overriding the extension point you can simply override the default ma
     myapp.target=.
     ```
 
-1. In this custom template create a file named `nxserver/config/elasticsearch-doc-mapping.json` to override the mapping. You can create a file named `nxserver/config/elasticsearch-doc-settings.json` to override the settings.<br/>
-**Important**: You must add your custom mapping to the existing one, you cannot just set your custom mapping in the file, Nuxeo does not merge your mapping with the default one. So, you must _duplicate_ the original file at `${NUXEO_HOME}/templates/common-base/nxserver/config/elasticsearch-doc-mapping.json` to `myapp/nxserver/config/elasticsearch-doc-mapping.json` , and modify the copy.
-
+1. In this custom template create a file named `nxserver/config/opensearch1-doc-mapping.json` to override the mapping. You can create a file named `nxserver/config/opensearch1-doc-settings.json.nxftl` to override the settings.<br/>
```

Comment:
There 's something confusing about path, aren't we missing some `{NUXEO_HOME}/templates/opensearch1-search-client` prefix?